src/payment/authorize-net-direct-post.md

```diff
@@ -5,103 +5,122 @@ title: Authorize.Net Direct Post - Deprecated
 {:.bs-callout-warning}
 **Deprecation Notice** <br/>
 Due to the Payment Service Directive [PSD2]({% link stores/compliance-payment-services-directive.md %}) and the continued evolution of many APIs, this payment integration is at risk of becoming outdated and no longer security compliant in the future. Additionally, Authorize.Net has deprecated the Authorize.Net Direct Post payment method. We are recommending that you disable it in your Magento configuration and transition to the Authorize.Net [Magento Marketplace extension](https://marketplace.magento.com/catalogsearch/result/?q=authorize.net){:target="_blank"}.<br/><br/>
-**This integration will be removed from the Magento 2.4.0 release and has been deprecated from current versions of 2.3.**<br/><br/>
+**This integration is removed from the Magento 2.4.0 release and has been deprecated from current versions of 2.3.**<br/><br/>
 For details about making a secure transition from deprecated payment integrations, see our [DevBlog](https://community.magento.com/t5/Magento-DevBlog/Deprecation-of-Magento-core-payment-integrations/ba-p/426445){:target="_blank"}.
 
 Authorize.Net handles all steps in the transaction process, such as payment data collection, data submission, and response to the customer, while the customer remains in your store.
 
-## Customer Workflow
+## Customer workflow
 
-1. **Customer chooses payment method**. During checkout, the customer chooses Authorize.Net Direct Post (Deprecated) as the payment method.
+1. **Customer chooses payment method**. During checkout, the customer chooses Authorize.Net Direct Post as the payment method.
 
-1. **Customer submits the order**. The customer enters the credit card information, reviews the order, and taps the Place Order button.
+1. **Customer submits the order**. The customer enters the credit card information, reviews the order, and clicks/taps the _Place Order_ button.
```

Comment:
Should we use the `<span class="btn"/>` syntax here?

Reply:
Because we are describing the workflow for the customer and not what the merchant is actually clicking, I thought that would not be the best match. The bold and button stylings are reserved for things the merchant must "touch" to complete a task.